log/logtest/recorder_test.go

```diff
@@ -68,17 +68,17 @@ func TestRecorderLoggerCreatesNewStruct(t *testing.T) {
 
 func TestLoggerEnabled(t *testing.T) {
 	for _, tt := range []struct {
-		name                   string
-		options                []Option
-		ctx                    context.Context
-		buildEnabledParameters func() log.EnabledParameters
+		name               string
+		options            []Option
+		ctx                context.Context
+		buildEnabledParams func() log.EnabledParameters
```

Comment:
nit: no need for a function here anymore.

```suggestion
		buildEnabledParams log.EnabledParameters
```

Reply:
Nice catch! I will address it before merging.

Reply:
PTAL https://github.com/open-telemetry/opentelemetry-go/pull/6009/commits/a3f93f168432d9d9ee6dabc210e7f3d79dfd06b7